sqlglot/expressions.py

```diff
@@ -450,6 +450,10 @@ class DateStrToDate(Func):
     pass
 
 
+class ExtractStruct(Func):
```

Comment:
should we call it StructExtract to keep it inline with duckdb?

Reply:
yes, let's do it